index.html

```diff
@@ -2529,6 +2531,85 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="emphasis">
+			<rdef>emphasis</rdef>
+			<div class="role-description">
+				<p>One or more emphasized characters. See related <rref>strong</rref>.</p>
+				<p>The <code>emphasis</code> role is intended to be used to indicate stress or emphasis of its contents; not for typographical presentation for presentation's sake. In general, authors SHOULD use this role only if the absence of the emphasis would change the meaning of the content.</p>
```

Comment:
A nit, but a role does not have content. An element that has a role applied to it has content. Since a role does recast, we could say an "emphasis element". But, I think we can simplify the wording without that.

Also, do we want to say "in general" at the start of a normative statement? If there are valid reasons for using emphasis to indicate purely presentational typography, then we should remove the normative statement.

Suggested wording:

>The purpose of the emphasis role is to stress or emphasize content. It is not for communicating changes in typographical presentation that do not impact the meaning of the content. Authors SHOULD use the emphasis role only if its absence would change the meaning of the content.